.github/workflows/windows.yml

```diff
@@ -0,0 +1,144 @@
+name: Windows build
+
+on:
```

Comment:
Should we have run on pull requests?

Reply:
Not sure because it's > 45 mins per run (!). Nightly?

Reply:
Let's see how many complaints we get...